src/components/UniversalNavbarExpanded/CtaButton.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+// EDS core components
+import { Button } from '../index'
+
+/**
+ * Simple CTA button for use in UniversalNavBarExpanded.
+ *
+ * TODO Add onKeyPress for tracking function
```

Comment:
Non-blocking, we survived this far without it but should still be added at some point.